scss/pattern-lib.scss

```diff
@@ -1,12 +1,2 @@
-@import 'vanilla-framework/scss/vanilla';
-@include vanilla;
-
 @import 'docs-vanilla-theme/scss/theme';
-@include docs-content;
-@include docs-footer;
-@include docs-header;
-@include docs-nav;
-@include docs-notes;
-@include docs-rows;
-@include docs-wrapper;
-@include docs-toc;
+@include docs-vanilla-theme;
```

Comment:
@anthonydillon One version of..? I should be including `vanilla-framework`, and then augment it with `docs-vanilla-theme`